Test/docker-compose.yml

```
@@ -61,14 +61,38 @@ services:
       target: dep-runner
     env_file:
       - ../Config/server.sample.env
+    # expose:
+    #   - "54000"
     ports:
-      - "50020:54000"
+      - "54000:54000"
```

When using default setting: `WithEnvoy: false`, I think that this line should not be changed.

```suggestion
      - "50020:54000"
```